yt_dlp/extractor/youtube.py

```diff
@@ -3621,7 +3648,7 @@ def _extract_player_response(self, client, video_id, master_ytcfg, player_ytcfg,
 
     def _get_requested_clients(self, url, smuggled_data):
         requested_clients = []
-        default = ['ios', 'android', 'web']
+        default = ['ios', 'web']
```

Comment:
guessing this was done for testing purposes and accidentally left in
```suggestion
        default = ['ios', 'android', 'web']
```